### PR TITLE
Windows and linux fixes

### PR DESCRIPTION
**Pull request by acnebs**

### CODE REVIEW

**Review by piggydoughnut — COMMENTED** [review]
The min/max/close controls should also appear on the login screen. Currently I can only see them on the Dashboard.

File: src/vars/styles/global.css [hunk 1]
```diff
@@ -5,16 +5,15 @@ body {
   transition: 0.2s;
   margin: 0;
   padding: 0;
-  -webkit-app-region: drag;
```
Comment (piggydoughnut) [inline src/vars/styles/global.css:8]: Afaik you cant drag the app around if you remove this.

File: src/components/Home/WindowControls.tsx [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+`;
+
+interface WindowButtonProps {
+  control: string;
+}
+
+const WindowButton = styled.button<WindowButtonProps>`
```
[the remark this answers is not in the document]
Comment (acnebs) [reply src/components/Home/WindowControls.tsx:22]: Yeah I figured for now we'd just go with the simple method, can add more native controls later.

File: src/vars/styles/global.css (hunk 1 above)
Comment (acnebs) [reply src/vars/styles/global.css:8]: You should be able to drag the app around by the top bar area.

**Pull request merged**